main.js

```diff
@@ -0,0 +1,66 @@
+
+var createPost = function(){
+  var obj = {}
+  obj.postName = $('#name-input').val();
+  obj.bodyText = $('#post-input').val();
+  $('#post-input').val('');
+  $('#name-input').val('');
+
+  var source = $('#post-template').html();
+  var template = Handlebars.compile(source);
+  var newHtml = template(obj);
+
+  $('#posts').append(newHtml);
+  };
+
+  $('#post-button').click(function() {
+    createPost();
+
+    //looping through multiple times when multiple posts are open.
+    $('.form-group').unbind('click').click(function(e) {
```

Comment:
I'm guessing `.unbind('click').click` is something from Stack Overflow?

Reply:
Also, `.form-group` is a generic bootstrap class for styling - I wouldn't target it with events as it's not very descriptive of what's happening here. Doing something like:

```js
$('.create-comment').click
```

Is self documenting